### ISSUE TITLE
Cannot save a multiscene animation - imagio:ffmpeg warning

### ISSUE THREAD

**Issue by ColinDuff** [pytroll/satpy#679]
Testing the pytrolls_example exercise7 notebook created for eumetsat training session i was unable to save then MSG animation created during the example.
Initailly i had to set the client option to false in save_animation as there was an error generated stating that the client object didnt have a gather attribute. After this change a save was attempted but not completed
```[DEBUG: 2019-03-26 15:58:57 : trollimage.xrimage] Applying gamma 1.5

[DEBUG: 2019-03-26 15:58:59 : native_msg] Calibration time 0:00:00.049262

[DEBUG: 2019-03-26 15:58:59 : satpy.writers] Enhancement configuration options: [{'name': 'linear_stretch', 'method': <function stretch at 0x7f9258516ea0>, 'kwargs': {'stretch': 'crude', 'min_stretch': 0.0, 'max_stretch': 100.0}}, {'name': 'gamma', 'method': <function gamma at 0x7f9258516c80>, 'kwargs': {'gamma': 1.5}}]

[DEBUG: 2019-03-26 15:58:59 : trollimage.xrimage] Applying stretch crude with parameters {'min_stretch': 0.0, 'max_stretch': 100.0}

[DEBUG: 2019-03-26 15:58:59 : trollimage.xrimage] Applying gamma 1.5

[WARNING: 2019-03-26 15:59:04 : imageio_ffmpeg] We had to kill ffmpeg to stop it. ```


**To Reproduce**
Run exercise 7 notebook



```

**Expected behavior**
The animation shuold have been  saved as an mp4 file

**Actual results**
File not saved

imagio version
imageio                   2.5.0                    py36_0    conda-forge
imageio-ffmpeg            0.2.0                      py_0    conda-forge

**Additional context**
imageio was downgraded to 2.4.1 and the notebook worked as expected

**Comment by djhoese** [comment]
How big was the area of the data being saved (rows x cols)?

**Comment by ColinDuff** [comment]
a full MSG image 3712 by 3712

**Comment by mraspaud** [comment]
Did the animation PR solve this ?

**Comment by ColinDuff** [comment]
well this and installing imagio 2.4.1 - works fine now

**Comment by djhoese** [comment]
This seems like a bug in ffmpeg. I'm guessing it is either not working with the newest versions of ffmpeg or more likely it is not communicating in the same way. I'm not sure there is an option for it but imageio probably needs to wait longer (be more patient) for ffmpeg to process our large frames. Either way it'll take a lot of testing.

**Comment by djhoese** [comment]
This should be fixed in `imageio-ffmpeg` versions 0.3+. I made a PR [here](https://github.com/imageio/imageio-ffmpeg/pull/14) that adds a `ffmpeg_timeout` flag and the default was changed from 2.5 seconds to 20 seconds. I'm closing this as it should be fixed now with the new version of `imageio-ffmpeg`.
